### DISCUSSION TITLE
Helferlöhne auf Dedi-Server

### DISCUSSION THREAD

**Discussion by Kotlett** [Courseplay/Courseplay_FS22#1204] [General]
Hallo und sorry, falls die Frage schon gestellt wurde, aber ich habe nichts dazu gefunden:
Wie verhält es sich mit den Helfer-Lohneinstellungen auf einem Dedi-Server? Den entsprechenden Menüeintrag können scheinbar, alle Spieler sehen - können Sie auch die Einstellungen ändern und speichern? 
Weil eigentlich macht es doch viel mehr Sinn, wenn diese Einstellung dem Server Admin vorbehalten bleibt? Ist dies irgendwie umsetzbar (ich habe in einem ersten Ansatz alle Optionen außer 100% aus dem Menü per LUA-Eingriff gelöscht, aber nun scheint er die Helfer nur mit 50% zu berechnen und das ist sicherlich nicht der beste Workaround ;)).

Vielen Dank!

**Comment by Tensuko** [comment]
Wir werden kein Admin only access draus machen.
Wenn man seinen Mitspielern nicht vertrauen kann, sehe ich das Problem eher da.

**Reply by Kotlett** [reply]
Alles klar, dann weiß ich aber dass das so beabsichtigt ist =)
